contracts/EthStorageContractL2.sol

```diff
@@ -35,55 +35,23 @@ contract EthStorageContractL2 is EthStorageContract2 {
         uint256 _dcfFactor
     ) EthStorageContract2(_config, _startTime, _storageCost, _dcfFactor) {}
 
+    /// @notice Get the current block number
+    function blockNumber() internal view override returns (uint256) {
+        return L1_BLOCK.number();
+    }
+
+    /// @notice Get the current block timestamp
+    function blockTs() internal view override returns (uint256) {
+        return L1_BLOCK.timestamp();
+    }
+
     /// @notice Get the randao value from the L1 blockhash.
-    function _getRandao(uint256 _l1BlockNumber, bytes calldata _headerRlpBytes) internal view returns (bytes32) {
+    function getRandao(
```

Comment:
Since we are using forge and hardhat at the same time, let's change the formatter to forge together (vscode => settings => solidity:formatter => forge)